ui/components/component-library/checkbox/checkbox.scss

```diff
@@ -0,0 +1,50 @@
+.mm-checkbox {
+  cursor: pointer;
+  -webkit-appearance: none;
+  appearance: none;
+  width: 20px;
+  height: 20px;
+
+  &:hover {
+    background-color: var(--color-background-default-hover);
+  }
+
+  &:focus {
+    border-color: var(--color-primary-default);
+  }
+
+  &__wrapper {
+    position: relative;
+  }
+
+  &__icon {
+    position: absolute;
+    top: 0;
+    left: 0;
+    pointer-events: none;
+  }
+
+  &--disabled {
+    opacity: var(--opacity-disabled);
+  }
```

Comment:
Hey @Akatori-Design, Do you we want to change the opacity of the label as well? Garrett and I think that the label should be reduced in opacity as well?